### ISSUE TITLE
Context of the Combine operation isn't to spec

### ISSUE THREAD

**Issue by brianpos** [HL7/fhirpath.js#150]
As per discussions:
https://chat.fhir.org/#narrow/stream/179166-implementers/topic/FHIRPath.20exclude
https://hl7.org/fhirpath/#combineother-collection-collection
The various fhirpath engines return for the below expression the context at the `t` trace to be:
* dotnet - the Patient
* fhirpath.js - the name
* HAPI - the given collection _(however Grahame has agreed that this is incorrect and that it should be Patient)_
* IBM - unclear, but think may be doing given like HAPI
```
name
.trace('fn')
.given
.combine(trace('t').family)
.trace('n')
.exclude('jim')
```

**Comment by plynchnlm** [comment]
name.given.combine(name.family) works, but curiously name.trace('n').given.combine(name.family) does not output the family names.

**Comment by yuriy-sedinkin** [comment]
Fixed in version 3.6.1.
